src/pages/currency-trading/forms/TransactionForm.js

```diff
@@ -108,6 +109,7 @@ export default function TransactionForm({ recordId, labels, maxAccess, plantId }
   const [idNumberOne, setIdNumber] = useState(null)
   const [search, setSearch] = useState(null)
   const [isClosed, setIsClosed] = useState(false)
+  const [fId, setFId] = useState(SystemFunction.CurrencyPurchase)
```

Comment:
why do we need this as state?

Reply:
yes here functionId change if sale or purchase